### ISSUE TITLE
"Preview of what I'm about to save" mode

### ISSUE THREAD

**Issue by jidanni** [zamojski/TowerCollector#146]
We recall that in an Ideal world,  cells are hexagons on the ground.

Okay, let's say we are on the ground and trying to map their boundaries.

Like with #144, we would like to see what we are about to save.

That way we can walk just up to the boundary,  save  that point, and then walk no further.

**Comment by zamojski** [comment]
To my it's a duplicate of #140.

**Comment by jidanni** [comment]
Actually they are slightly different.
One is like always showing the current cell.
One is only after we press some "Save Point" button, show
us what we are about to save, then we press "OK".
